packages/block-editor/src/components/list-view/use-list-view-expand-selected-item.js

```diff
@@ -27,12 +27,6 @@ export default function useListViewExpandSelectedItem( {
 		[ firstSelectedBlockClientId ]
 	);
 
-	const parentClientIds =
-		Array.isArray( selectedBlockParentClientIds ) &&
-		selectedBlockParentClientIds.length
-			? selectedBlockParentClientIds
-			: null;
-
```

Comment:
This has been removed so that we can use `selectedBlockParentClientIds` in the `useEffect` dependency array — in theory at least, `selectedBlockParentClientIds` should only be a new array when there really has been a change and we wish to perform an update, since it gets returned by the `useSelect` instead of being instantiated on each render.

Reply:
If `selectedBlockParentClientIds?.length` does the same job, LGTM to remove it :)